Pythonwin/pywin/idle/PyParse.py

```diff
@@ -122,6 +122,7 @@ def dump(*stuff):
     _tran[ord(ch)] = ")"
 for ch in "\"'\\\n#":
     _tran[ord(ch)] = ch
+# TODO
 # We are called with unicode strings, and str.translate is one of the few
 # py2k functions which can't 'do the right thing' - so take care to ensure
 # _tran is full of unicode...
```

Comment:
How should this be updated to remove python2 support code? Is it just about removing `_tran = "".join(_tran)` ?